### DISCUSSION TITLE
remark-footnotes vs prettier

### DISCUSSION THREAD

**Discussion by stefanprobst** [remarkjs/remark#680] [Q&A] [answered]
i am trying to align `remark-stringify` output with `prettier`'s formatting, which works great, but one issue with footnotes formatting i cannot figure out.

`prettier` with `proseWrap: "always"` formats long footnotes like this:

![prettier](https://user-images.githubusercontent.com/20753323/114262998-3b830400-99e3-11eb-984d-6e6aa25051b2.png)

`remark` formats long footnotes like this:

![remark](https://user-images.githubusercontent.com/20753323/114262985-302fd880-99e3-11eb-81c3-fc2349ef2f16.png)

is there a way to align this?

---

my config in `package.json`:

```json
{
  "scripts": {
    "format:prettier": "prettier . --write --ignore-path .gitignore",
    "format:remark": "remark . --output --ignore-path .gitignore"
  },
  "devDependencies": {
    "prettier": "^2.2.1",
    "remark-cli": "^9.0.0",
    "remark-footnotes": "^3.0.0",
    "remark-gfm": "^1.0.0"
  },
  "prettier": {
    "proseWrap": "always"
  },
  "remarkConfig": {
    "settings": {
      "bullet": "-",
      "emphasis": "_",
      "listItemIndent": "one",
      "rule": "-"
    },
    "plugins": [
      "remark-gfm",
      "remark-footnotes"
    ]
  }
}
```

**Comment by wooorm** [comment, marked as answer]
Both have slightly different preferences. Why not use both together?

**Reply by stefanprobst** [reply]
that's what i'll do!
